nuxeo-jsf/nuxeo-platform-ui-web/src/main/java/org/nuxeo/ecm/platform/ui/web/tag/fn/DocumentModelFunctions.java

```diff
@@ -906,7 +876,7 @@ public static String getLabelFromId(String directoryName, String id) {
         if (id == null) {
             return "";
         }
-        try (Session directory = getDirectoryService().open(directoryName)) {
+        try (Session directory = DirectoryHelper.getDirectoryService().open(directoryName)) {
```

Comment:
Call `Framework.getService(DirectoryService.class)` directly, no need for the helper.